source/sass/components/publication-page.scss

```diff
@@ -53,6 +53,11 @@
   height: 18px;
   margin-right: 10px;
   background-image: url(../_images/glyphs/download-file.svg);
+  transition: $button-icon-transition-speed;
```

Comment:
For code readability, let's nest these two blocks together & use the `hover-focus-active` mixin as we'd like the transition to kick in for focus/active state as well

```scss
.download-button {
  &::before {
    ...
    ...
    ...
  }

  @include hover-focus-active {
    &::before {
      filter: brightness(0);
    }
  }
}
```

Reply:
Hi Mavis,

Great suggestion! I have added it to the latest commit